pkg/httpclient/json.go

```diff
@@ -178,6 +179,17 @@ func (c *Client) Stream(method, path string, in, out interface{}) (stream.Stream
 	return c.StreamWithHeader(method, path, make(http.Header), in, out)
 }
 
+func (c *Client) ResumingStream(method, path string, ch interface{}) (stream.Stream, error) {
```

Comment:
I think I prefer `ResumingStream` to indicate it is resuming from the last ID, rather than just reconnecting. Will leave as is, unless your opinions strengthen.